<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonBleConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Light off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonToggle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Blinky</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonScan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan Peripheral Devices</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetBleDevices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelBleConnectionStatus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserDeviceAttributes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBluetooth_LE_Console">
    <property name="title">
     <string>Bluetooth LE Console</string>
    </property>
   </widget>
   <addaction name="menuBluetooth_LE_Console"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
